pkg/channel/consolidated/reconciler/controller/kafkachannel.go

```diff
@@ -165,7 +165,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, kc *v1beta1.KafkaChannel
 		return r.kafkaConfigError
 	}
 
-	kafkaClusterAdmin, err := r.createClient()
+	kafkaClusterAdmin, err := r.createClient(logger)
```

Comment:
More than passing logger, can you pass the context? Same below, it's generally nicer to pass context, since it can contain other stuff you have in the signature.

Reply:
Sounds good, will do